fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -61,16 +61,13 @@ def execute(command, hide_output)
           rescue Errno::EIO
             # Exception ignored intentionally.
             # https://stackoverflow.com/questions/10238298/ruby-on-linux-pty-goes-away-without-eof-raises-errnoeio
-          ensure
-            Process.wait(pid)
           end
         end
       rescue => ex
         @errors << ex.to_s
       end
 
-      exit_status = $?.exitstatus
-      unless exit_status.zero?
+      if exit_status != 0
```

Comment:
Is there a reason for not keeping the user of `.zero?` here? (and the other spot below)

Reply:
It is functionally equivalent but its a better way of making sure that we are comparing an number to a number. I would personally keep the `.zero?` 😊